<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWidget</class>
 <widget class="QDialog" name="SettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>251</width>
    <height>207</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <widget class="QCheckBox" name="checkBoxCityToIp">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Get city to IP</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>City or zip-code:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineCity">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>54</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Style:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxStyle">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Dark</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Light</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Update every </string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxMinutes">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>45</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>54</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>minutes</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBoxOkCancel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>176</width>
     <height>27</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>checkBoxCityToIp</tabstop>
  <tabstop>lineCity</tabstop>
  <tabstop>spinBoxMinutes</tabstop>
  <tabstop>comboBoxStyle</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
